packages/daemon/src/directory.js

```diff
@@ -97,6 +97,30 @@ export const makeDirectoryMaker = ({
       return formatLocator(id, formulaType);
     };
 
+    /** @type {import('./types.js').EndoDirectory['reverseLocate']} */
+    const reverseLocate = async locator => {
```

Comment:
Silly idea:
- “locate” means “get locator for name”
- “nominate” means “get name from locator”